src/Adapter/Module/ModuleDataUpdater.php

```diff
@@ -106,10 +106,14 @@ public function upgrade($name)
             if (LegacyModule::initUpgradeModule($module)) {
                 $legacy_instance = LegacyModule::getInstanceByName($name);
                 $legacy_instance->runUpgradeModule();
-
-                LegacyModule::upgradeModuleVersion($name, $module->version);
-
-                return !count($legacy_instance->getErrors());
+                
+                if (empty($legacy_instance->getErrors()){
+                    LegacyModule::upgradeModuleVersion($name, $module->version);
+                    Hook::exec('actionModuleUpgradeAfter', array('object' => $module));
```

Comment:
```suggestion
                    Hook::exec('actionModuleUpgradeAfter', array('module' => $module));
```
Suggestion: maybe simply provide a key that describes the content of this parameter ? :)

Reply:
Sure why not? 2 (or more) brains are better than one 💯